src/SILHS/version_silhs.txt

```diff
@@ -0,0 +1,288 @@
+commit 46cac2787c3802be0418762948ad6d1231dde182
```

Comment:
Likewise, we won't need to commit this file because it's automatically generated.